bundle-workflow/tests/test_perf_workflow/test_perf_cluster.py

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+import unittest
+from unittest.mock import MagicMock, patch
+
+from manifests.bundle_manifest import BundleManifest
+from test_workflow.perf_test_cluster import PerformanceTestCluster
+
+
+class TestPerformanceCluster(unittest.TestCase):
+    def setUp(self):
+        os.chdir(os.path.dirname(__file__))
```

Comment:
This shouldn't be needed, has side effects. Constructs paths relative to `dirname(__file__)` and then load the manifest from there.